connectors/services/job_scheduling.py

```diff
@@ -39,6 +39,7 @@ def __init__(self, config):
         self.source_list = config["sources"]
         self.connector_index = None
         self.sync_job_index = None
+        self.last_wake_up_time = datetime.utcnow()
```

Comment:
What this means, if that if server starts 1 second after something should be scheduled, it won't be scheduled. It's probably fine, but we can also do `datetime.utcnow() - timedelta(seconds=IDLING)` to still execute jobs that should have started IDLING seconds ago